modules/rosa-hcp-vpc-manual.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc
+// * rosa_getting_started/quickstart.adoc
+
+:_content-type: PREFERENCE
+[id="rosa-hcp-vpc-manual_{context}"]
+=== Creating a Virtual Private Cloud (VPC) manually
+
+To manually create your VPC, go to link:https://us-east-1.console.aws.amazon.com/vpc/[the VPC page in the AWS console]. Your VPC must have the following details.
+
+.Requirements for your VPC
+[options="header",cols="50,50"]
+|===
+| Requirement | Details
```

Comment:
* Check the "requirement" column for capitalization. I know "Availability Zone" has both in caps in AWS docs too, but afaik, it should be "VPC name", "Public subnet", "S3 VPC endpoint" and "DNS hostname"
* In the "details" column, these should all probably be complete sentences, or, if not, be parallel to each other in construction. 
* What does `-` mean in the endpoint?